charts/tuf/README.md

```diff
@@ -54,6 +54,7 @@ A framework for securing software update systems - the scaffolding implementatio
 | secrets.rekor.name | string | `"rekor-public-key"` |  |
 | secrets.rekor.path | string | `"rekor.pub"` |  |
 | secrets.tsa.create | bool | `false` |  |
+| secrets.tsa.existingSecret | bool | `false` |  |
```

Comment:
Why not make this `enabled` and set as `true`. If `secrets.tsa.create` is `true`, a new secret will be created. Otherwise, `secrets.tsa.name` is the name of the existing secret

Reply:
I implemented the changes. I thought it would be better to set it as `enabled` as `false` by default since the Charts are independent, but I could change it if you consider the other way.

Reply:
I like this approach a lot! It would be great if we can implement the same pattern across all of the secrets. Though that probably requires a separate PR to avoid encompassing too much into this issue. Unless you would want to rename the PR for that scope

Reply:
As the TSA case, I set the default value to `false` to be independent.

Reply:
@sabre1041 Any news on this?

Reply:
@cvegagimenez syntactically this does work. However, in practice, for tuf to run properly, it will need at least one source of content (a secret) in order to start properly. Should we enforce that at least one secret is provided?

Also, would you be able to resolve the conflict in the `README.md` file?

Reply:
Done. I also added the same checks for the other TUF objects.